components/monitors/cu_consolemon/src/lib.rs

```diff
@@ -664,7 +669,17 @@ fn init_error_hooks() {
     }));
 }
 
+fn setup_terminal() {
+    enable_raw_mode()
+        .map_err(|e| CuError::new_with_cause("Could not enable console raw mode", e))
+        .expect("Could not enter raw mode: check terminal compatibility.");
```

Comment:
Do we need both map_err and expect here? I think the latter would do

Reply:
oh yeah, indeed a brainless copy paste / autocomplete line lol